client_start_charge.go

```diff
@@ -65,9 +62,9 @@ func (c *Client) StartChargePoints(
 	if httpResponse.StatusCode != http.StatusOK {
 		return nil, newStatusError(httpResponse)
 	}
-	var response StartChargePointsResponse
+	var response StartChargeResponse
 	if err := json.NewDecoder(httpResponse.Body).Decode(&response); err != nil {
 		return nil, err
 	}
 	return &response, nil
-}
\ No newline at end of file
+}
```

Comment:
I saw now that the line is not here?! Will make add a new line and update it tomorrow!

Reply:
I think it is here! otherwise there would be a :stop_sign: here instead